raiden/constants.py

```diff
@@ -3,8 +3,10 @@
 
 from eth_utils import keccak, to_checksum_address, to_hex
 
+from raiden.utils.typing import RaidenProtocolVersion
+
 SQLITE_MIN_REQUIRED_VERSION = (3, 9, 0)
-PROTOCOL_VERSION = 1
+PROTOCOL_VERSION = RaidenProtocolVersion(1)
```

Comment:
`RaidenProtocolVersion` is failing the tests because of circular dependencies in raiden/utils/__init__.py

Reply:
grrr, I'm removing this for now. To fix this I would have to move everything from `raiden.utils.__init__` somewhere else.